docs/ops/pooling/AdaptiveMaxPool_8.md

```diff
@@ -0,0 +1,87 @@
+## AdaptiveMaxPool<a name="AdaptiveMaxPool"></a> {#openvino_docs_ops_pooling_AdaptiveMaxPool_8}
+
+**Versioned name**: *AdaptiveMaxPool-8*
+
+**Category**: *Pooling*
+
+**Short description**: Applies max pooling with adaptive kernel size over the input.
+
+**Detailed description**: This operation calculates the output based on the 1st input and `output_size` determined by the 2nd input.
+The kernel dimensions are calculated using the following formulae for the `NCDHW` input case:
+
+\f[
+\begin{array}{lcl}
+d_{start} &=& floor(i*D_{in}/D_{out})\\
+d_{end}   &=& ceil((i+1)*D_{in}/D_{out})\\
+h_{start} &=& floor(j*H_{in}/H_{out})\\
+h_{end}   &=& ceil((j+1)*H_{in}/H_{out})\\
+w_{start} &=& floor(k*W_{in}/W_{out})\\
+w_{end}   &=& ceil((k+1)*W_{in}/W_{out})
+\end{array}
+\f]
+
+The output is calculated following this formula:
+
+\f[
+Output(i,j,k) = max(Input[d_{start}:d_{end}, h_{start}:h_{end}, w_{start}:w_{end}])
+\f]
+
+**Attributes**:
+
+*   *output_type*
+
+  * **Description**: the output tensor type
+  * **Range of values**: "i64" or "i32"
+  * **Type**: string
+  * **Default value**: "i64"
+  * **Required**: *No*
+
+**Inputs**:
+
+*   **1**: 3D, 4D or 5D input tensor of shape `[N,C,L]`, `[N,C,H,W]` or `[N,C,D,H,W]` and type *T*. Required.
```

Comment:
Please use consistent naming for layouts (at least for these two operations). Here you use `[N,C,L]` layout for 3d tensors while for avg pool you use `[N,C,W]`